office-backend/package.json

```diff
@@ -9,15 +9,16 @@
   },
   "keywords": [],
   "author": "",
-  "license": "EUPL-1.2+",
+  "license": "ISC",
   "description": "",
-  "dependencies": {
-    "fastify": "5.2.2"
-  },
   "devDependencies": {
-    "@types/node": "22.13.16",
-    "ts-node": "10.9.2",
-    "ts-node-dev": "2.0.0",
-    "typescript": "5.8.2"
+    "@types/node": "^22.13.13",
```

Comment:
we should NOT use greater than (prefixed with `^`) versions
AND not go down in version

Reply:
It is automatically removed by Renovate itself too I believe. :-) Renovates creates PRs whenever we accidentally add a ^ to remove it again I think.

We have no ^'s at all in our package.json as far as I know?

Reply:
Ah sorry, look at the ZAC project.. You are right, all ^'s should be removed. We use exact versions only.